src/style.css

```diff
@@ -1,3 +1,197 @@
 body {
   margin: 0;
+  font-family: "Roboto", sans-serif;
+}
+
+p {
+  margin: 0;
+}
+
+a {
+  text-decoration: none;
+}
+
+.card {
+  margin: auto;
+  margin-top: 25px;
+  width: 198px;
+  background-color: #fff;
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+}
+
+.card__img {
+  width: 160px;
+  height: 134px;
+
+  padding-top: 32px;
+  padding-left: 19px;
+}
+
+img {
+  width: 100%;
+  height: 100%;
+}
+
+.card__text {
+  width: 166px;
```

Comment:
```suggestion

```
It's a bad practice to hardcode the width of the content, especially if it has 100% of the container width on the Figma design. It's better to limit it with padding of the card.